src/ck-ldb/tree_level.ci

```diff
@@ -0,0 +1,35 @@
+module tree_level {
+
+  message LBStatsMsg_1 {
+    int pe_ids[];
+    float bgloads[];
+    float speeds[];
+    unsigned int obj_start[];
+    float oloads[];
+    unsigned int order[];
+  };
+
+  message LLBMigrateMsg {
+    int num_incoming[];
+    int obj_start[];
+    int to_pes[];
+  };
+
+  message SubtreeLoadMsg {
+  };
+
+  message SubtreeMigrateDecisionMsg {
+    // FIXME? I don't think this layout is ideal, but I don't know if ci files
```

Comment:
I think we do support arrays of any PUPable type inside messages, and it should be just as efficient as integral types so long as the type is declared as PUPbytes